dts/bindings/sensor/phosense,xbr818.yaml

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Phosense XBR818 I2C-capable 10Ghz Radar Sensor
```

nit: use proper convention for International System of Units

```suggestion
    Phosense XBR818 I2C-capable 10 Ghz Radar Sensor
```

---

Done.